javascript/bower.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "ola",
+  "version": "0.0.1",
```

Comment:
I wonder if the version of this should be tied to the OLA version? Where does it show up?